submit-plan/action.yml

```diff
@@ -106,6 +113,7 @@ runs:
             $tf_plan_output_arg \
             --log '${{ inputs.log }}' \
             ${{ inputs.plan-json }} \
+            $app_arg \
```

Comment:
I don't think that the go args parser is so strict, but _I_ would prefer all options before all arguments.

Reply:
moved it before the inputs.plan-json